CocoaGrapes/ViewController.swift

```diff
@@ -7,13 +7,27 @@
 //
 
 import UIKit
+import SwiftGifOrigin
+import MarqueeLabel
 
 class ViewController: UIViewController {
 
     @IBOutlet weak var imageView: UIImageView!
-
+    @IBOutlet weak var marqueeLabel: MarqueeLabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        //Gif
+        let theDankness = UIImage.gif(name: "dankgrapes")
+        imageView.image = theDankness
+        
+        //Marquee Label
+        marqueeLabel.animationCurve = .easeInOut
+        marqueeLabel.tapToScroll = true
+        marqueeLabel.speed = MarqueeLabel.SpeedLimit.rate(30.0)
```

Comment:
nit: you can actually just do `.rate(30.0)` here, without the `MarqueeLabel.SpeedLimit`
alternatively you can set the rate directly `marqueeLabel.rate = 30`